test/fixtures/recordings/Integration-test-for-up-command_3614339512/up-command-delivers-correct-response-_668345128/recording.har

```diff
@@ -37,98 +37,78 @@
               "value": "with params"
             }
           ],
-          "url": "https://adobeioruntime.net/api/v1/web/helix/default/embed/https://embeds.project-helix.io/jekyll_embed?params=with%20params"
+          "url": "https://adobeioruntime.net/api/v1/web/helix/helix-services/embed@v1https://embeds.project-helix.io/jekyll_embed?params=with%20params"
```

Comment:
@trieloff it looks like something messes up the path to the action....

Reply:
Yes, the old URL is quite outdated.

Reply:
well, I mean, there is a `/` missing after the `@v1`